src/components/Product-card/product-card.jsx

```diff
@@ -0,0 +1,44 @@
+import "../../pages/Product-listing/product-listing.css";
+
+const ProductCard = ({ productcard }) => {
+  const { title, price, rating, categoryImg } = productcard;
+
+  return (
+    <>
+    <div className="card-image">
+      <div className="card-vertical">
+        <div className="card-vertical-info flex-column">
+          <img
+            className="img-responsive adjust-image"
+            src={categoryImg}
+            alt="shopping-item"
+          />
+          <p className="card-text">{title}</p>
+
+          <div className="rating-badge">
+            {rating}
+            <span className="star-icon">
+              <i class="fa-solid fa-star"></i>
+            </span>
+          </div>
+
+          {/* <p className="card-subtext">
+              A bit more detail about the product
```

Comment:
remove unwanted comments
keep only meaningful comments

Reply:
Thank you, I might use this later so I've kept it.